README.md

```diff
@@ -186,23 +188,25 @@ strategies:
   "RemovePodsViolatingNodeTaints":
     enabled: true
 ````
+
 ## Pod Evictions
 
-When the descheduler decides to evict pods from a node, it employs following general mechanism:
+When the descheduler decides to evict pods from a node, it employs the following general mechanism.
 
 * [Critical pods](https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/) (with priorityClassName set to system-cluster-critical or system-node-critical) are never evicted.
-* Pods (static or mirrored pods or stand alone pods) not part of an RC, RS, Deployment or Jobs are
+* Pods (static or mirrored pods or stand alone pods) not part of an RC, RS, Deployment or Job are
 never evicted because these pods won't be recreated.
 * Pods associated with DaemonSets are never evicted.
 * Pods with local storage are never evicted.
-* Best efforts pods are evicted before Burstable and Guaranteed pods.
-* All types of pods with annotation descheduler.alpha.kubernetes.io/evict are evicted. This 
-annotation is used to override checks which prevent eviction and user can select which pod is evicted. 
-User should know how and if the pod will be recreated.
+* Best efforts pods are evicted before burstable and guaranteed pods.
+* All types of pods with the annotation descheduler.alpha.kubernetes.io/evict are evicted. This 
+annotation is used to override checks which prevent eviction and users can select which pod is evicted. 
```

Comment:
can you indent the line so it aligns with `All types of pods`?

Reply:
Fixed